templates/webapi/main/dashboard_build_results.html.ep

```diff
@@ -6,6 +6,7 @@
     % if (@{$groupresults->{children}}) {
         <h2>
             %= link_to $group->{name} => url_for('parent_group_overview', groupid => $group->{id})
+            %= include 'main/test_result_overview_link', group_list => \@{$groupresults->{children}}, groupid => $group->{id}
```

Comment:
Dereferencing the array ref via `@{…}` just to take the reference again via `\` makes no sense.
```suggestion
            %= include 'main/test_result_overview_link', group_list => $groupresults->{children}, groupid => $group->{id}
```

Reply:
done, thanks